tests/web/karma.conf.js

```diff
@@ -22,13 +26,12 @@ module.exports = function(config) {
       {pattern: 'third-party/node_modules/chai/chai.js'},
       {pattern: 'source/web/d3/d3.min.js'},
       {pattern: 'source/web/d3/d3-tip.min.js'},
-      {pattern: 'tests/web/test_header.js'},
       {pattern: 'tests/web/assets/lineage-example.json', included: false},
```

Comment:
> > RE: I'm a little unsure about getting rid of test_widget.html. Having html files like that around for manual debugging is useful. I agree that it's technically unnecessary, though
> > Fair. I'm in favor of getting rid of it because it's confusing for it to be there (to me) and not actually used, but I don't have a super strong opinion on this.
> 
> I think that's reasonable. People can make their own files like this easily enough

Ah, I see the quote reply feature now! I definitely think that HTML examples should exist in the examples directory, though.

Reply:
100% agree! I think that's really future-proof.